src/cdk/menu/menu-base.ts

```diff
@@ -123,6 +123,13 @@ export abstract class CdkMenuBase
   focusFirstItem(focusOrigin: FocusOrigin = 'program') {
     this.keyManager.setFocusOrigin(focusOrigin);
     this.keyManager.setFirstItemActive();
+
+    // If there's no active item at this point, it means that all the items are disabled.
+    // Move focus to the menuPanel panel so keyboard events like Escape and closing child menus using arrow keys still works.
+    // Also, this will give _some_ feedback to screen readers.
+    if (!this.nativeElement.contains(document.activeElement)) {
```

Comment:
Instead of going to the DOM, I think that we can check `keyManager.activeItem === null` here.

Reply:
Sorry for the delayed response, I tried it but didn't work, the `keyManager.activeItem` at this point is the _child menu trigger button_.